### ISSUE TITLE
Convergence trend of spectrum loss

### ISSUE THREAD

**Issue by RongerSnow** [cc-hpc-itwm/UpConv#10]
Thank you for exploring this interesting constraint. In a similar way to yours, I also add spectrum loss to my generated network, but it seems that its value fluctuates all the time. Unlike MSE, it doesn't have a significant downward convergence trend. I am quiet confused whether this is a normal phenomenon? It would be better if you could provide your training log for reference.

**Comment by RicardDurall** [comment]
Based on our experiments, we have also observed that the spectrum loss in some scenarios can fluctuate. However, as with many constraints, it is a matter of fine-tuning the hyper-parameters. 
On our supplemental material, we provide a training log plot of the AE case. There, you can see how exists a certain downward convergence trend (especially in the beginning) and then it flattens. I hope I have helped you.
